validation/validation.go

```diff
@@ -346,6 +346,11 @@ func validateUPluginJSON(archive *zip.Reader, uPluginFile *zip.File, withValidat
 					Path: file.Name,
 					Type: "sml_mod",
 				})
+			} else if extension == "so" {
+				modInfo.Objects = append(modInfo.Objects, ModObject{
+					Path: file.Name,
+					Type: "sml_mod",
+				})
```

Comment:
Merge this into the if statement above with `extension == "dll" || extension == "so"`

Reply:
https://github.com/satisfactorymodding/smr-api/pull/12/commits/c92ca90e3a9e65269f3fa50c8ff3d9637e571008